check_manifest.py

```diff
@@ -605,6 +604,18 @@ def read_config():
         IGNORE_BAD_IDEAS.extend(p for p in patterns if p)
 
 
+def _find_config(config):
+    """Search setup.cfg and tox.ini for the section '[check-manifest]'.
+    If successful, the file will have been read into the config object."""
+    search_files = ['setup.cfg', 'tox.ini']
+    for filename in search_files:
+        if (os.path.isfile(filename)
+            and config.read([filename])
+                and config.has_section(CFG_SECTION_CHECK_MANIFEST)):
+                return True
```

Comment:
ConfigParser ignores files that do not exist without raising errors, so this can be simplified to

```suggestion
        if (config.read([filename])
                and config.has_section(CFG_SECTION_CHECK_MANIFEST)):
            return True
```

(Yeah, indentation here is giving me a headache; no matter what I try it feels wrong ;).  But I don't want `return True` to be indented 8 spaces relative to the `if`.)

Reply:
Oh, yeah, the `os` check is definitely redundant. And agreed.